xlators/mgmt/glusterd/src/glusterd.h

```diff
@@ -417,8 +397,7 @@ struct glusterd_volinfo_ {
     int op_version;
     int client_op_version;
     int32_t quota_xattr_version;
-    pthread_mutex_t reflock;
-    int refcnt;
+    gf_atomic_t refcnt;
```

Comment:
This is a 64bit atomic, the previous refcnt was 32bit?

Reply:
Hm... 'gf_atomic_t' is commonly used nearly everywhere and I don't see a reason to enforce 'gf_atomic_int32t'  unless there are thousands of objects of the given type.